packages/tocco-ui/src/ButtonGroup/ButtonGroup.stories.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {storiesOf} from '@storybook/react'
+import {boolean, select, withKnobs} from '@storybook/addon-knobs'
+
+import ButtonGroup from './'
+import Button from '../Button/Button'
+import ButtonLink from '../ButtonLink/ButtonLink'
+import {stylingInk, stylingLook} from '../utilStyles'
+
+storiesOf('Navigation', module)
+  .addDecorator(withKnobs)
+  .add(
+    'ButtonGroup',
+    () => (
+      <ButtonGroup
+        ink={select('ink', stylingInk)}
+        look={select('look', stylingLook)}
```

Comment:
please orient yourself by the button example. This look knob does not work properly (wrong default value and you can choose values that are not allow by proptype)